docker-compose.yml

```diff
@@ -6,13 +6,13 @@ services:
       POSTGRES_USER: dev
       POSTGRES_PASSWORD: dev
     volumes:
-      - dustvolume:/postgres
+      - pgsql:/var/lib/postgresql/data
```

Comment:
I tried running the following to move the existing files to this new path:

```
docker run --rm -it \
  --volume dustvolume:/postgres \
  --volume dustvolume:/var/lib/postgresql/data \
  alpine sh -c "cp -a /postgres/. /var/lib/postgresql/data/"
 ```
 
 It yields:
 ```
cp: '/postgres/.' and '/var/lib/postgresql/data/.' are the same file
```